src/components/ImagesCarousel.js

```diff
@@ -130,8 +130,10 @@ export const ImagesCarousel = ({
 
   if (!data || data.length === 0) return null;
 
+  const sliderSettings = data[0]?.sliderSettings || {};
```

Comment:
`sliderSettings`? in your example in the PR description there ist `settings`.
please explain the new "first element of data" logic more explicitly and how this affect the `carouselData`

Reply:
I updated the code but forgot to send a commit. The code is up to date now.